apax/md/ase_calc.py

```diff
@@ -95,6 +95,7 @@ def unpack_results(results, inputs):
     unpacked_results = []
     for i in range(n_structures):
         single_results = jax.tree_map(lambda x: x[i], results)
+        single_results["energy"] = single_results["energy"].item()
         for k, v in single_results.items():
```

Comment:
wasn't this already in main, maybe merge main into this branch first?

Reply:
yeah well, you independently fixed it while the PR was open :sweat_smile: